src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,11 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        ColorSupplier colorSupplier = new ColorSupplier();
+        NumberSupplier numberSupplier = new NumberSupplier();
+        Lottery randomBall = new Lottery(colorSupplier, numberSupplier);
+        for (int i = 0; i < 3; i++) {
```

Comment:
Consider replacing the magic number '3' with a named constant to improve code readability and maintainability. For example, private static final int NUMBER_OF_BALLS = 3; at the class level, and then use NUMBER_OF_BALLS in the loop.